core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -685,9 +685,20 @@ fd_table_insert_existing(struct fd_table *ft, __wasi_fd_t in, int out)
     struct fd_object *fo;
     __wasi_errno_t error;
 
-    if (fd_determine_type_rights(out, &type, &rights_base, &rights_inheriting)
-        != 0)
+    error =
+        fd_determine_type_rights(out, &type, &rights_base, &rights_inheriting);
+    if (error != 0) {
+#ifdef BH_PLATFORM_EGO
```

Comment:
How about renaming it to `BH_PLATFORM_OPEN_ENCLAVE`? I remember you mentioned that the lower platform is Open Enclave:
https://github.com/openenclave/openenclave

Reply:
The specification is from EGO not Open Enclave. Make it as `BH_PLATFORM_OPEN_ENCLAVE` will be not appropriate.

Reply:
OK, got it.